packages/@aws-cdk/aws-cloudtrail/test/cloudtrail.test.ts

```diff
@@ -625,19 +625,77 @@ describe('cloudtrail', () => {
         });
       });
 
-      test('managementEvents set to None correctly turns off management events', () => {
+      test('not provided and managementEvents set to None throws', () => {
```

Comment:
For this test can we specify the error instead of just "throws", i.e. using `toThrowError()`. I think the exception is `InvalidEventSelectorsException`, but whatever the error we expect is.

Reply:
Is it just the description you want updated? As far as I understand, exceptions thrown using `node.addValidation(...)` use a generic `throw new Error(...)` rather than a custom exception class.